src/apps/service/apps/combined/util.clj

```diff
@@ -16,7 +16,7 @@
   "Expects results to be a list of maps in a format like {:total int, :apps []}"
   [params results]
   (let [params      (apply-default-search-params params)
-        maybe-deref (fn [r] (if (future? r) @r r))
```

Comment:
When I was testing, I encountered an error in this function because `r` was `nil` one time. I haven't been able to duplicate it since. This should fix the problem, but I can't be sure since I haven't been able to reproduce the problem.